app/controllers/eventos_controller.rb

```diff
@@ -0,0 +1,50 @@
+class EventosController < ApplicationController
+  #before_action :authenticate_usuario!, except: [:index, :show]
+  #before_action -> { redireciona_usuario(:pode_postar_evento?) }, except: [:index, :show]
+
+    def index
+        @eventos = Evento.order(:data_inicio)
+    end
+
+    def new
+        @evento = Evento.new
+    end
+
+    def create
+        @evento = Evento.new(evento_params)
+        if @evento.save
+            flash[:success] = "Evento criado"
+            redirect_to edit_evento_path(@evento)
+        else
+            flash[:danger] = "Falha ao criar evento"
+            render :new
+    end
+  end
+
```

Comment:
Parece que indentação tá errada nessas primeiras linha. Tá tudo pra direita de onde deveria.
To usando sempre 2 espaços pra indentar.